src/style.css

```diff
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+  --star--size: 16px;
+}
+
+.container {
+  width: 96px;
+  height: 96px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: var(--star--size);
+  height: var(--star--size);
+  background-image: url("../src/images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n) {
+  background-image: url("../src/images/star-active.svg");
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url("../src/images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url("../src/images/star-active.svg");
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url("../src/images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("../src/images/star-active.svg");
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url("../src/images/star-active.svg");
+}
```

Comment:
I think you can simplify it like this:
```css
.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
  background-image: url(./images/star-active.svg);
}
```
hope It will work